### ISSUE TITLE
Search in the Block Manager component should announce the search results

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#15498]
The search in the Block Manager modal:

<img width="360" alt="Screenshot 2019-05-08 at 09 56 01" src="https://user-images.githubusercontent.com/1682452/57376008-3ab21980-719f-11e9-9e87-e0a0976fdeb3.png">

doesn't announce the search results with an audible feedback. All the similar searches, including the one in the main inserter, do announce the search results with a `speak()` message. The Block Manager search should do that as well.

**Comment by draganescu** [comment]
Hi @afercia, by "announce" do you mean this message in the inserter:

```
const resultsFoundMessage = sprintf(
			/* translators: %d: number of results. */
			_n( '%d result found.', '%d results found.', resultCount ),
			resultCount
		);
		debouncedSpeak( resultsFoundMessage );
```

... which announces how many results are found? Or is it something else?

**Comment by afercia** [comment]
Hi @draganescu. Yes it's the `a11y` package `speak()` message. In the example you provided, it comes from the  `withSpokenMessages` Higher Order Component and it's the debounced version.
